checklists/azure_arc_checklist.en.json

```diff
@@ -190,11 +190,11 @@
             "category": "Management and Monitoring",
             "subcategory": "Management",
             "waf": "Operations",
-            "text": "Consider using Azure Automanage to control settings and avoid configuration drift on servers",
-            "description": "Azure Automanage help implement Microsoft best-practices for servers management in Azure",
+            "text": "Consider using Azure Machine Configuration to control settings and avoid configuration drift on servers",
+            "description": "Azure Machine Configuration to help implement Microsoft best-practices for servers management in Azure",
             "guid": "7a927c39-74d1-4102-aac6-aae01e6a84de",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/azure/automanage/automanage-arc"
+            "link": "https://learn.microsoft.com/en-us/azure/governance/machine-configuration/overview"
```

Comment:
Hey could you please remove the localization (`en-us/`) from the link?

Reply:
Resolved!